### DISCUSSION TITLE
B站部分XHR请求没有被ajax hook代理

### DISCUSSION THREAD

**Discussion by DeltaFlyerW** [the1812/Bilibili-Evolved#1799] [提问 / Q&A]
![image](https://user-images.githubusercontent.com/48078918/116714257-26a8f780-aa08-11eb-945c-c3f7f15feb6f.png)
B站评论区获取数据的api,如https://api.bilibili.com/x/v2/reply/main?callback=jQuery17205289213983282228_1619794752740&jsonp=jsonp&next=0&type=1&oid=15680827&mode=2&plat=1&_=1619794803730,如图所示,并没有被ajax hook代理.

**Comment by the1812** [comment]
我在普通视频页试了下是没问题的
![image](https://user-images.githubusercontent.com/26504152/116767837-3f93c600-aa65-11eb-833b-464f1797c55f.png)
